dark-facebook-messenger-theme.css

```diff
@@ -30,8 +34,8 @@
 }
 
 /* `administator` text for people in groups and a lot more texts */
-._1ht7, ._497p, ._2v6o, ._3eus, ._1lj0, ._kmc ._1p1t, ._1n-e._36zg, ._1n-e, ._5qsj {
-    color: rgba(234, 234, 234, 0.4);
+._1ht7, ._497p, ._2v6o, ._3eus, ._1lj0, ._kmc ._1p1t, ._1n-e._36zg, ._1n-e, ._5qsj, ._58al::placeholder, ._14-7._58al::-moz-placeholder, ._14-7._58al:-moz-placeholder {
+    color: rgba(234, 234, 234, 0.4) !important;
```

Comment:
Can use `/* csslint allow: known-properties, important */` to allow `!important` for `CSSLint`